pkg/resources/custom_diffs.go

```diff
@@ -0,0 +1,45 @@
+package resources
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/internal/provider"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/customdiff"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func NestedIntValueAccountObjectComputedIf(key string, parameter sdk.AccountParameter) schema.CustomizeDiffFunc {
```

Comment:
I don't think we can, but I'll try.

Reply:
Somehow created the tests for those customdiffs, but that's some black magic

Reply:
They look good to me :)

Just one thought (maybe for the next PR or even for the future): the setups for the four tests in `TestNestedValueComputedIf` are really similar: what value is in the state, what value is in the config, what return value is expected. We could improve readability in this case by introducing a common setup function (wrapping the map ogic), changing the calculateDiff, or parametrizing (in this case it would be helpful).

But this is just food for thought.